tagstudio/src/core/library/alchemy/library.py

```diff
@@ -91,6 +91,51 @@ def get_default_tags() -> tuple[Tag, ...]:
     return archive_tag, favorite_tag
 
 
+def get_default_colors(namespace: ColorNamespace) -> list[Color]:
+    colors: list[Color] = [
+        #TODO: reduce this to limited tagstudio standard color set
```

Comment:
I don't know if you have settled on a final list of colors you do want in the default pack, but the sooner I can have that the better

Reply:
No problem, I'll get that fleshed out for you